vendor/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "vendor",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "postinstall": "./postinstall.sh"
+  },
+  "devDependencies": {
+    "code-oss-dev": "cdr/vscode#96a09a7846538c3bbedb6a2aeca729537bb8202b"
```

Comment:
Can this be in the root package.json?

Reply:
Unfortunately not for a few reasons. We don’t want the `postinstall` scripts to run because they’re made for already built and released package which differs in shape than the development version. This will also be installed without the dev dependencies which we for our own build process. Lastly, we work out of `vendor/modules` because their Webpack configuration can’t discern between `code-server/node_modules/` and `code-server/node_modules/vscode/node_modules/`, so it assumes that its own dependencies are not accessible. Anything short of a privileged chroot makes this really difficult to overcome. 

-  has webpack

Reply:
@jsjoeio ^

We can follow up with further documentation after #4136

Reply:
Ahh neato. Makes sense!